scripts/tool_shed/build_ts_whoosh_index.py

```diff
@@ -6,7 +6,9 @@
 
 Make sure you adjusted your config to:
  * turn on searching via toolshed_search_on
- * specify toolshed_whoosh_index_dir where the indexes will be placed
+ * specify whoosh_index_dir where the indexes will be placed
+
+Also make sure that GALAXY_EGGS_PATH variable is properly set.
```

Comment:
It works without setting GALAXY_EGGS_PATH at all if invoked as above from the root, right?